staging/src/k8s.io/client-go/tools/cache/reflector_test.go

```diff
@@ -1753,6 +1756,152 @@ func TestReflectorListExtract(t *testing.T) {
 	}
 }
 
+func TestReflectorWithUnsafeDelete(t *testing.T) {
```

Comment:
what is it testing? The name should better be a claim, not a feature description.

Reply:
`TestReflectorRecoversWithRightRVOnUnsafeDelete` or something like that.